libcontainer/specconv/example.go

```diff
@@ -200,8 +200,14 @@ func ToRootless(spec *specs.Spec) {
 	// Fix up mounts.
 	var mounts []specs.Mount
 	for _, mount := range spec.Mounts {
-		// Ignore all mounts that are under /sys.
-		if strings.HasPrefix(mount.Destination, "/sys") {
+		// Replace the /sys mount with an rbind.
+		if mount.Destination == "/sys" {
```

Comment:
Needs filepath.Clean ?

Reply:
According to https://github.com/opencontainers/runtime-spec/blob/master/config.md, `mount.Destination` MUST be an absolute path.

We should validate that the destination is absolute elsewhere. Alas we do not do that. Will open a PR soon.

Reply:
Filed https://github.com/opencontainers/runc/pull/2917 but realized we are talking about clean path, not absolute path here.

Fixed.